doc/functions/attribute.rst

```diff
@@ -26,11 +35,6 @@ attribute:
 
     {{ attribute(object, method) is defined ? 'Method exists' : 'Method does not exist' }}
 
-.. note::
-
-    The resolution algorithm is the same as the one used for the ``.``
-    operator, except that the item can be any valid expression.
```

Comment:
I would keep the note saying it is the same algorithm than the dot operator (as it avoids having to duplicate the doc to explain which algorithm is used by this function), removing only the end of the sentence.

Reply:
Done while merging.